obj/objects.cpp

```diff
@@ -199,7 +202,7 @@ bool TriangleObj::doesHit(const core::Vec3 &point) const {
   float beta = vecC.crossProduct(vecA).magnitude() / 2;
   float gamma = vecA.crossProduct(vecB).magnitude() / 2;
 
-  return alpha + beta + gamma - area_ <= constants::kHitAccuracy;
+  return alpha + beta + gamma - area_ <= constants::kAccuracy;
```

Comment:
You realize that now you use the same constant to compare completely different quantities.

Time vs area vs distance (I am guessing you have it somewhere).

Reply:
Do I really need another variable to determine accuracy for only triangle calss? I added comment:
![image](https://user-images.githubusercontent.com/45213495/99908971-f9804c80-2ce5-11eb-831c-6f1a0e09548d.png)

I hope this is enough. I will have to implement test to test if its right tho ;)